frontend/src/components/buttons/LastRunCheckbox.styles.js

```diff
@@ -9,11 +9,11 @@ export const StyledDiv = styled.div`
     display: flex;
     flex-direction: ${props => props.direction};
     border: 1px solid var(--tonic-grey);
-    border-radius: 4px;
+    border-radius: var(--space-4);
     width: ${props => (props.direction === 'column' ? '10%' : '30%')};
     max-width: 230px;
     min-width: ${props => (props.direction === 'column' ? '200px' : '230px')};
-    line-height: 34px;
+    line-height: 30px;
```

Comment:
A bit more refinement on the checkbox styles would be nice...
![image](https://user-images.githubusercontent.com/17247485/111777250-b9a82b00-88bb-11eb-8bb4-65998fd76606.png)

Reply:
This was actually fixed in other PR.